src/code_80091750.c

```diff
@@ -1889,20 +1889,20 @@ void func_80094A64(s32 arg0) {
     func_80092290(5, &D_8018E854, &D_8018E85C);
     func_80099A70();
     func_8009C918();
-    temp_t1 = D_800E86A0;
+    temp_t1 = gMenuSelection;
```

Comment:
First time making a PR for this project... I wasn't sure if I should be simplifying logic where appropriate (like removing the extra temp_t1 variable here)

Reply:
I'm guessing I probably shouldn't modify any logic of the generated C code unless I get the entire function matching the assembly?

Reply:
You're right that in-this case the temp_t1 doesn't exist. I wouldn't change these unless you were working on that function trying to match it. In-progress matches are fine. (So feel free to change things). But if you're using `./diff func_name`  you'll know if  something worked better or not.